extensions/package-vulnerability-scanner/src/components/ContentList.vue

```diff
@@ -98,6 +103,24 @@ const filteredContent = computed(() => {
 
   return scannerStore.content;
 });
+
+const userHeader = computed(() => {
+  let name;
+
+  if (props.user.first_name && props.user.last_name) {
+    name = `${props.user.first_name} ${props.user.last_name}`;
+  } else if (props.user.first_name) {
+    name = props.user.first_name;
+  } else if (props.user.last_name) {
+    name = props.user.last_name;
+  }
+
+  if (name === undefined) {
+    name = props.user.username;
+  }
+
+  return `${name}'s Connect Content`;
```

Comment:
Ditto from previous PRs, what about encapsulating this within the store to something like `userStore.headerDisplayLabel` ?

Reply:
Good call 👍

Reply:
Ah if I do this I'm realizing that I'll have to handle `undefined` and I lose the guaranteed nature of using `prop.user`. I'll leave this as is, but if I need the user's name in a display format in other places in the future I'll move it to the store.